docker/irods_clients/ubuntu/Dockerfile

```diff
@@ -121,13 +129,15 @@ RUN curl -sSL https://packages.irods.org/irods-signing-key.asc | apt-key add - &
     tee /etc/apt/sources.list.d/renci-irods.list && \
     apt-get update && \ 
     apt-get install -q -y --no-install-recommends \
+    gnuplot \
```

Comment:
Does this mean we can drop it from deployments in OS? I think we install it with conda or apt :S

Reply:
I really don't think it's a good idea to couple your production environment setup to whatever is configured here, which is a general-purpose client image that's being used mainly in Docker Compose setups for development.
If an application (and I'm including the DD in that definition) needs a Docker image, it should have it's own self-managed Dockerfile for that, rather than extending its requirements into a general-purpose image.

Reply:
I added gnuplot here so that the scripts that come with samtools in the container work. It is not intended to make `gnuplot` generally available from this contianer (and so `gnuplot` is not in the manifest).

Reply:
The issue still stands wrt other behaviours of the image. Just as long as you know that other factors e.g. the need to test a new version of baton may mean `latest` of this image jumps to a new version without coordinating with a DD release.

Reply:
Understand that we're streching the use of this container building beyond orginal purpose and remit.
Do we need a story to remove samtools et al. from this image to be built into separate image (with iRODS support)? Or are there other ways forward?  DD will only use a particular version/release of the containers produced - so it is "safe" but I think (flip side of the concern you @kjsanger  voice?) we need to be wary of deadlocking releases (this repo, or DD) by binding too tightly.